float64[] new_frame_prob
bool termination
---
string action_seq
bool success
